src/functions/tarsUpload/domain/upload/TARSPayloadConverter.ts

```diff
@@ -103,4 +105,33 @@ export class TARSPayloadConverter implements ITARSPayloadConverter {
       passCertificate: passCompletion.passCertificateNumber,
     };
   }
+
+  private populateAdi3Mark(
+      completedTestPayload: CompletedTestPayload,
+      testType: number|undefined,
+      test: TestResultSchemasUnion,
+  ): CompletedTestPayload {
+    if (testType !== 11) {
```

Comment:
Could we maybe check here for ADI3 being the testCategory instead? 11 is ambiguous if you don't follow the code through